controls/SV-244553.rb

```diff
@@ -1,56 +1,69 @@
 control 'SV-244553' do
-  title "RHEL 8 must ignore IPv4 Internet Control Message Protocol (ICMP)
-redirect messages."
-  desc  "ICMP redirect messages are used by routers to inform hosts that a more
-direct route exists for a particular destination. These messages modify the
-host's route table and are unauthenticated. An illicit ICMP redirect message
-could result in a man-in-the-middle attack."
-  desc  'rationale', ''
-  desc  'check', "
-    Verify RHEL 8 ignores IPv4 ICMP redirect messages.
+  title 'RHEL 8 must ignore IPv4 Internet Control Message Protocol (ICMP)
+redirect messages.'
+  desc "ICMP redirect messages are used by routers to inform hosts that a more direct route exists for a particular destination. These messages modify the host's route table and are unauthenticated. An illicit ICMP redirect message could result in a man-in-the-middle attack.
```

Comment:
It seems odd to me that this requirement doesn't have the same router caveat as the other simular controls, this may be a tech writter error. Let's see if we can verify that this should be in the same class from older guidance documents.